homeassistant/components/synology_dsm/camera.py

```diff
@@ -104,7 +108,7 @@ def device_info(self) -> DeviceInfo:
             identifiers={
                 (
                     DOMAIN,
-                    f"{self._api.information.serial}_{self.camera_data.id}",
+                    f"{self._api.information.serial}_{self.config_entry_id}_{self.camera_data.id}",
```

Comment:
You can't just change unique identifiers. This will generate duplicate devices for everyone. This will need a migration and cannot be put in a patch release.

Reply:
I've changed the wrong string🤦 will correct this